macros/bool_or.sql

```diff
@@ -0,0 +1,23 @@
+{% macro bool_or() %}
```

Comment:
Not very strongly opinionated on this but it feels like having arg like `field` would be a more logical API.

```SQL
{{ bool_or(field) }}
#rather than
{{ bool_or() }}(field)
```